models/serialize.js

```diff
@@ -0,0 +1,34 @@
+let serializer = (attributes) => {
+  return (models) => {
+    if (!Array.isArray(models)) {
+      models = [models];
+    }
+
+    let data = models.map((model) => {
+      return {
+        "type": model.key.kind,
+        "id": model.key.id,
+        "attributes": attributes.reduce((previous, attribute) => {
```

Comment:
Ok so I assume we are using `reduce` here to be flexible. While there's only one attribute right now for building the data field in the payload, this could grow.